apps/web/src/modules/create-dao/components/AllocationForm/AllocationForm.tsx

```diff
@@ -159,7 +164,9 @@ export const AllocationForm: React.FC<AllocationFormProps> = ({ title }) => {
         )}
       </Formik>
 
-      <ContributionAllocation />
+      {(chain.id === CHAIN_ID.ETHEREUM || chain.id === CHAIN_ID.GOERLI) && (
```

Comment:
nit: could be worth pulling out as a const that is designated to mainnet ETH and goerli ETH

Reply:
this is the only place we have specific logic for mainnet + goerli ETH